src/api/app/jobs/send_event_emails.rb

```diff
@@ -13,24 +8,45 @@ def perform
         # if someone else saved it too, better don't continue - we're not alone
         return false
       end
-      subscribers = event.subscribers
-      next if subscribers.empty?
-      EventMailer.event(subscribers, event).deliver_now
-      create_rss_notifications(event)
+
+      subscriptions = event.subscriptions
+      next if subscriptions.empty?
+
+      # 1. Send the emails to the instant_email subscribers
+      subscribers_instant_email = subscriptions.select(&:instant_email?).map(&:subscriber)
+      EventMailer.event(subscribers_instant_email, event).deliver_now if subscribers_instant_email.any?
+
+
+      # 2. Create rss notifications for the instant_email subscribers
+      subscriptions.select(&:instant_email?).each do |subscription|
```

Comment:
Right ok, I can add a method like that in, but since all notifications are either instant_emails or daily_emails, the method may as well be like this:
```ruby
def rss_feed?
  true
end
```